### PR TITLE
Fixed the CORS/load balancer issue with the Ray Assistant

### PR DESCRIPTION
**Pull request by GokuMohandas**
## Why are these changes needed?

New services doesn't give the CORS issue or load balancer 503 issue.

### CODE REVIEW

File: doc/source/_static/js/assistant.js [hunk 1]
```diff
@@ -139,7 +139,7 @@ function rayAssistant(event) {
 
         async function readStream() {
             try {
-                const response = await fetch('https://ray-assistant-public-bxauk.cld-kvedzwag2qa8i5bj.s.anyscaleuserdata.com/chat', {
+                const response = await fetch('https://ray-assistant-public-98zsh.cld-kvedzwag2qa8i5bj.s.anyscaleuserdata.com/chat', {
```
Comment (aslonnie) [inline doc/source/_static/js/assistant.js:142]: why is this deployed as a new service now? why the domain changes?

this means past versions of ray docs will no longer be working?

**Review by angelinalg — APPROVED** [review]
Stamp

File: doc/source/_static/js/assistant.js (hunk 1 above)
Comment (aslonnie) [reply doc/source/_static/js/assistant.js:142]: @KamenShah ?

Comment (KamenShah) [reply doc/source/_static/js/assistant.js:142]: I'm working with AWS to understand why the load balancer is returning 503s and load balancer logs are not helpful.

Comment (GokuMohandas) [reply doc/source/_static/js/assistant.js:142]: yup the previous link has the load balancer issues that Kamen is looking into and this new service (with no changes to the actual application) works just fine.

**Pull request closed without merging**